lib/insights.ts

```diff
@@ -75,21 +75,21 @@ class AlgoliaAnalytics {
   // Public methods
   public init: (params: InitParams) => void;
   public initSearch: (params: InitSearchParams) => void;
-  public clickedObjectIDsAfterSearch: (params?: InsightsSearchClickEvent) => void;
+  public clickedObjectIDsAfterSearch: (
+    params?: InsightsSearchClickEvent
+  ) => void;
   public clickedObjectIDs: (params?: InsightsClickObjectIDsEvent) => void;
   public clickedFilters: (params?: InsightsClickFiltersEvent) => void;
   public convertedObjectIDsAfterSearch: (
     params?: InsightsSearchConversionEvent
```

Comment:
Is there a reason why the names `InsightsSearchConversionEvent` and `InsightsConversionObjectIDsEvent` are not consistent (not the `ObjectIDs` part)? It seems that for the others events the name is consistent.

Reply:
would you expect something like `InsightsConversionSearchObjectIDsEvent`?

Search always use objectIDs, it can be considered as implied by the `Search` part

Reply:
One or the other (doesn't matter) but we should be consistent.